### ISSUE TITLE
merge neighborhoods44.json into crosswalk output

### ISSUE THREAD

**Issue by jotok** [DCActionforChildren/dcaction#155]
This is currently being done in the front-end code.

**Comment by cmgiven** [comment]
Yes, for additional detail here, note that `neighborhoods44.json` is our geojson file. It contains just a geo_id and an id, but no data. The `neighborhood.csv` file contains a row for each feature in the geojson, containing all of the data. By merging these files, we could save a file load and some front-end processing.
